src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
```suggestion
    public static Colors getRandomColor() {
        private final Random random = new Random();
        
        int index = random.nextInt(Colors.values().length);
```
lets make it a class-level variable - to avoid creating a new instance each time this method is called